ci-operator/step-registry/aws/provision/route53/private-hosted-zone/associate-with-vpc-in-different-account/aws-provision-route53-private-hosted-zone-associate-with-vpc-in-different-account-ref.metadata.json

```
@@ -0,0 +1,11 @@
+{
+	"path": "aws/provision/route53/private-hosted-zone/associate-with-vpc-in-different-account/aws-provision-route53-private-hosted-zone-associate-with-vpc-in-different-account-ref.yaml",
```

```suggestion
	"path": "aws/provision/route53/private-hosted-zone/associate-vpc-in-diff-account/aws-provision-route53-private-hosted-zone-associate-vpc-in-diff-account-ref.yaml",
```
For a shorter step name?